### ISSUE TITLE
stylelint incorrectly regards .toLocaleString()'s option as CSS-in-JS

### ISSUE THREAD

**Issue by SimonSimCity** [shinnn/vscode-stylelint#253]
This library is showing an error on the following line of code:

```js
(positionsChecked / scansCount).toLocaleString('de-DE', { style: 'percent' })
```

> Unexpected unknown property "style" (property-no-unknown)

**Comment by shinnn** [comment]
```diff
- This library
+ stylelint
```

Report this bug on https://github.com/stylelint/stylelint/issues instead.
